vivarium/interface/interactive.py

```diff
@@ -61,52 +86,75 @@ def take_steps(self, number_of_steps=1, step_size=None, with_logging=True):
                 self.step(step_size)
 
     def list_values(self):
+        if not self._setup:
```

Comment:
This is repeated enough times that it might be worth building a simple validation decorator. Something like:
```
def requires_setup(func):
  def inner(self, *args, **kwargs):
    if not self._setup:
      raise InteractiveError("Value pipeline configuration is not complete until the simulation is setup.")
￼    return func(self, *args, **kwargs)
  return inner
```